README.md

```diff
@@ -1,2 +1,26 @@
+
+One common approach to this problem is to hire or leverage dedicated research software engineers (RSEs) [cite Ritt Group, NYU, Princeton] or contract with data management/consulting companies (ex. DataJoint) to convert the data. But, since these resources tend to be distributed across many different universities in institutions, much of the work maybe duplicated. Common conversion frameworks and tools can help accelerate development and make conversions easier for all stakeholders.
+
+Just as labs frequently seek to convert backlogged data, they are also often motivated to establish conversion pipelines in advance of experimental data collection, so that newly generated data can be automatically converted into NWB with minimal duplicated work. For example, a lab may develop a conversion pipeline on existing data, and then try to generalize the pipeline so that it can be used for any follow up experiments. Automatically converting diverse neurophysiology data formats presents challenges due to the wide variety of modalities involved. Even a single lab may employ voltage recording, optical imaging, optogenetics, behavior, and more, each with its own software-dependent formats lacking standardization. A lab-specific conversion pipeline must take all the possible options into account to properly future-proof, or adapt itself to every new experiment that adds or removes a piece of software or hardware. These considerations make conversion pipelines using _pyNWB_ or _MatNWB_ alone difficult to develop and even harder to maintain, and are often cited as a specific pain point [cite A perspective on neuroscience data standardization with NWB].
+
+To mitigate these challenges and promote wider adoption of NWB, we have developed a sophisticated software suite comprising three core tools. The first, _NWB Inspector_, is a Python-based tool that ensures the validity of NWB files, enhancing the reliability of their metadata when uploading to sharing platforms like DANDI. The second, _NeuroConv_, is a software package that facilitates the conversion of neurophysiology data from various proprietary formats to the NWB standard.
+Its primary purpose is to make it easy to develop automatic conversion pipelines that ingest data from all recording and processing sources used in a neurophysiology experiment. The third tool, _NWB GUIDE (Graphical User Interface for Data Entry)_, is an interactive desktop application that walks users through the process of converting their data to the NWB standard without requiring any programming whatsoever.
```

Comment:
```suggestion
To mitigate these challenges and promote wider adoption of NWB, we have developed a sophisticated software suite comprising three core tools. The first, _NWB Inspector_, is a Python-based tool that ensures the validity of NWB files, enhancing the reliability of their metadata when uploading to sharing platforms like DANDI. The second, _NeuroConv_, is a software package that facilitates the conversion of neurophysiology data from various proprietary formats to the NWB standard. Its primary purpose is to make it easy to develop automatic conversion pipelines that ingest data from all recording and processing sources used in a neurophysiology experiment. The third tool, _NWB GUIDE (Graphical User Interface for Data Entry)_, is an interactive desktop application that walks users through the process of converting their data to the NWB standard without requiring any programming whatsoever.
```

Unless you'd prefer to have each sentence on a different line to allow different lines on the GitHub changelogs? If so should probably do it everywhere as habit

Reply:
Nope, one line per paragraph renders properly and looks nice to me. --fixed